client/src/components/home/index.tsx

```diff
@@ -378,6 +385,11 @@ function HomePage(props: {
           reviewAnswerState.clearError();
         }}
       />
+      <NotificationDialog
+        title={"This page is for setting up your mentor!"}
+        open={!hasSeenSplash}
+        closeDialog={() => userSawSplashScreen("")}
```

Comment:
This needs to pass an accessToken to the userSawSplashScreen function instead of the empty string